charts/gardener-extension-provider-gcp/templates/priorityclass.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: {{ include "priorityclassversion" . }}
+kind: PriorityClass
+metadata:
+  name: gardener-extension-provider-gcp
+value: 1000000000
```

Comment:
Same question as in https://github.com/gardener/gardener-extension-provider-azure/pull/181#discussion_r510122959

Reply:
reolved.